tests/testProjectClassToR1.m

```diff
@@ -0,0 +1,286 @@
+classdef testProjectClassToR1 < matlab.unittest.TestCase
+
+    properties
+        inputStandardProjectClass
+        inputStandardStruct
+        outputStandardStruct
+        defaultProject
+        outputCustomStruct
+        inputCustomProjectClass
+        modelFile
+        fullPath
+    end
+
+    methods(TestClassSetup)
+        function defineVariables(testCase)
+            testCase.inputStandardProjectClass = "originalDSPCBilayerProjectClass.mat";
+            testCase.inputStandardStruct = "originalDSPCBilayerStructInput.mat";
+            testCase.outputStandardStruct = "originalDSPCBilayerStructOutput.mat";
+            testCase.defaultProject = "defaultProject.mat";
+            testCase.outputCustomStruct = "monolayerVolumeModel.mat";
+            testCase.inputCustomProjectClass = "monolayerVolumeModelProjectClass.mat";
+            testCase.modelFile = "monolayerVolumeCustomLayerModel.m";
+        end
+    end
+
+    methods(TestMethodSetup)
+        function setWorkingFolder(testCase)
+            root = getappdata(0,'root');
+            testCase.fullPath = [root filesep 'tests' filesep 'testProjectClassToR1Conversion'];
+            import matlab.unittest.fixtures.CurrentFolderFixture
+            testCase.applyFixture(CurrentFolderFixture(testCase.fullPath))
+        end
+    end 
+
+     methods(TestMethodTeardown)
+        function testCleanUp(testCase)
+            cd (testCase.fullPath)
```

Comment:
If you use a temp directory then you can avoid this it will be deleted by the fixture